services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,41 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+  if (xQueueSend(thermalMgrQueueHandle, event, 1) != pdTRUE) return ERR_CODE_QUEUE_FULL; // timeout?
 
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
   /* Implement this function */
+  thermal_mgr_event_t event;
+  event.type = THERMAL_MGR_EVENT_INTERRUPT;
+  if (thermalMgrSendEvent(&event) != ERR_CODE_SUCCESS) ; // What should be done if error?
```

Comment:
we don't have any way to log the error in the onboarding currently from an ISR, but in our actual code base you would use `LOG_ERROR_CODE_FROM_ISR`